internal/pkg/cli/pipeline_init.go

```diff
@@ -137,43 +144,49 @@ func newInitPipelineOpts(vars initPipelineVars) (*initPipelineOpts, error) {
 		sel:              selector.NewSelect(prompter, ssmStore),
 		runner:           exec.NewCmd(),
 		fs:               &afero.Afero{Fs: afero.NewOsFs()},
+		wsAppName:        wsAppName,
 	}, nil
 }
 
-// Validate returns an error if the flag values passed by the user are invalid.
+// Validate returns an error if the optional flag values passed by the user are invalid.
 func (o *initPipelineOpts) Validate() error {
-	if o.appName == "" {
+	// This command must be executed in the app's workspace because the pipeline manifest and buildspec will be created and stored.
+	if o.wsAppName == "" {
```

Comment:
Can we reuse `validateInputApp` this function? Also for the previously merged PR for `pipeline` deploy sorry...

Reply:
Ooh, yes-- I hadn't seen that! Thanks!